Game/ui/asteroid/asteroid.gd

```diff
@@ -91,87 +92,68 @@ func _init_ores_for_each_player() -> Dictionary[int, Array]:
 	for player_id in ConnectionSystem.get_player_id_list():
 		ores_for_each_player[player_id] = []
 	return ores_for_each_player
-
-
-func _in_same_board(pos1: TileMapPosition, pos2: TileMapPosition) -> bool:
-	if pos1 and pos2:
-		return pos1.player_id == pos2.player_id
-	else:
-		return false
-
-
-func _get_tile_map_pos() -> TileMapPosition:
-	for player_id in ConnectionSystem.get_player_id_list():
-		var tile_map := _get_tile_map(player_id)
-		if tile_map.mouse_inside_tile_map():
-			var tile_position := tile_map.get_mouse_tile_map_coords()
-			return TileMapPosition.new(player_id, tile_position)
+	
+
+## Returns the grid coordinates the mouse is over
+func _get_new_building_coordinates() -> TileMapPosition:
```

Comment:
Since Tilemaps are gone, this class should probably be renamed.

Reply:
I would like some sort of class that groups all player boards into a single object that can be queried for things like "what cell is the mouse hovering over" so we can just deal with the cells directly rather than handling TileMapPosition. We needed TileMapPosition before because we needed to deal with tilemaps not being able to pick out particular cells, but that is not a limiting factor anymore.